### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by No-1l** [Empyreann/discord-dms-opener#5] [Q&A]
Hello, i did the steps, and after i launch "python main.py"
i get this error : 
![image](https://user-images.githubusercontent.com/129275441/228506401-41567de8-20e6-4f45-a46c-c6505a2743aa.png)

Any help please ?

**Comment by arnavgpt** [comment]
@No-1l Can you show your `config.json` file?

Everything in the `config.json` file should be "strings"
